streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -271,11 +265,13 @@ public void shouldClassifyExistingTasksWithStateUpdater() {
 
         taskManager.handleAssignment(standbyTasks, restoringActiveTasks);
 
-        Mockito.verify(stateUpdater).getTasks();
-        Mockito.verify(stateUpdater).remove(standbyTaskToRecycle.id());
-        Mockito.verify(stateUpdater).remove(standbyTaskToClose.id());
-        Mockito.verify(stateUpdater).remove(restoringActiveTaskToRecycle.id());
-        Mockito.verify(stateUpdater).remove(restoringActiveTaskToClose.id());
+        verify(stateUpdater).getTasks();
+        verify(stateUpdater).remove(standbyTaskToRecycle.id());
+        verify(stateUpdater).remove(standbyTaskToClose.id());
+        verify(stateUpdater).remove(restoringActiveTaskToRecycle.id());
+        verify(stateUpdater).remove(restoringActiveTaskToClose.id());
+
+        verifyConsumerResumedWithAssignment(consumer);
```

Comment:
There are several tests where we are adding this verification, could you elaborate a bit why?

Reply:
Same as above.